catalystwan/api/templates/feature_template.py

```diff
@@ -122,16 +122,12 @@ def get(cls, session: ManagerSession, name: str) -> FeatureTemplate:
 
         feature_template_model = choose_model(type_value=template_info.template_type)
 
-        device_specific_variables: Dict[str, DeviceVariable] = {}
-        values_from_template_definition = find_template_values(
-            template_definition_as_dict, device_specific_variables=device_specific_variables
-        )
+        values_from_template_definition = find_template_values(template_definition_as_dict)
```

Comment:
Have we tested how FeatureTemplates behave with this change? Current serialization (generate_payload) code is a bit messy. I'm not sure if it will work well with it.

Reply:
Current feature templates tests are passing